elements/rh-tile/rh-tile-group.ts

```diff
@@ -0,0 +1,184 @@
+import { LitElement, html, type PropertyValueMap } from 'lit';
+import { classMap } from 'lit/directives/class-map.js';
+import { customElement } from 'lit/decorators/custom-element.js';
+import { property } from 'lit/decorators/property.js';
+import { colorContextConsumer, type ColorTheme } from '../../lib/context/color/consumer.js';
+import { colorContextProvider, type ColorPalette } from '../../lib/context/color/provider.js';
+import { getRandomId } from '@patternfly/pfe-core/functions/random.js';
+import { InternalsController } from '@patternfly/pfe-core/controllers/internals-controller.js';
+import { RovingTabindexController } from '@patternfly/pfe-core/controllers/roving-tabindex-controller.js';
+import { RhTile, TileSelectEvent } from './rh-tile.js';
+
+import styles from './rh-tile-group.css';
+
+/**
+ * A group of `<rh-tile>` elements which handles radio selection.
+ * @slot - tiles
+ */
+@customElement('rh-tile-group')
+export class RhTileGroup extends LitElement {
+  static readonly styles = [styles];
+
+  /**
+   * whether tile interaction is disabled but retains ability to be read by screen readers;
+   * preferred method of disabling instead of `disabled`
+   */
+  @property({ reflect: true, attribute: 'aria-disabled', type: String }) ariaDisabled = 'false';
+
+  /**
+   * if tile is checkable, whether only on tile can be checked
```

Comment:
```suggestion
   * if tile is checkable, whether only one tile can be checked
```

what about `<rh-tile checkable="radio">` instead of `<rh-tile checkable radio>`?